test/common_interface/arkode.jl

```diff
@@ -21,11 +21,13 @@ prob = SplitODEProblem(SplitFunction(f1, f2; analytic = (u0, p, t) -> exp(2t) *
 sol = solve(prob, ARKODE(; linear_solver = :Dense))
 @test sol.errors[:l2] < 1e-2
 
-sol = solve(prob,
-    ARKODE(; linear_solver = :LapackBand, jac_upper = 3, jac_lower = 3);
-    reltol = 1e-12,
-    abstol = 1e-12)
-@test sol.errors[:l2] < 1e-6
+@test_broken begin
```

Comment:
bad bot

Reply:
I told it to do this for now as all of the Lapack stuff is failing. I'm trying to see if things can work sans lapack, and coming back to BLAS/LAPACK issues later.